src/test/resources/application-test.yml

```diff
@@ -28,14 +28,6 @@ feature:
     enable-delius-search: true
     enable-nomis-search: true
 
-spring:
-  jpa:
-    properties:
-      hibernate:
-        hbm2ddl:
-          auto: update
-    show-sql: false
-
```

Comment:
Why do we need to remove this?

Reply:
This is in the test so not much to bother with, it is just that we need to manually do things like the one above.